internal/nginx-meshctl/commands/services.go

```diff
@@ -140,9 +140,7 @@ func isNamespaceInjectionEnabled(ctx context.Context, k8sClient client.Client, n
 		fmt.Printf("error getting namespace: %v\n", err)
 		return false, err
 	}
-	return slices.Contains(*meshConfig.EnabledNamespaces, ns) ||
-		nsObj.GetLabels()[mesh.AutoInjectLabel] == mesh.AutoInjectionEnabled ||
-		(*meshConfig.IsAutoInjectEnabled && !(mesh.IgnoredNamespaces[ns])), nil
```

Comment:
Since we're able to get rid of this we might be able to get rid of the `mesh.IgnoredNamespaces` constant entirely. Not sure, but do we have a defined intended behavior if a user labels an 'ignored namespace' for injection?

Reply:
If this var isn't used anywhere else, we can probably get rid of it. We do add the `nginx-mesh` namespace to the map at startup, so it usually contains `kube-system` and `nginx-mesh`.

But our MutatingWebhhok should filter out those namespaces, so this var may not have any value anymore.

Reply:
`IgnoredNamespaces` is an internal list of namespaces we never want to inject. The only ones we ignore are `kube-system` and the namespace where NSM is deployed. I believe we should leave this since we don't want to risk breaking k8s or the mesh. 

I verified that after adding the injection label to the `nginx-mesh` namespace and rerolled one of the deployments, the deployment was not injected. So we should be good there. I am updating the docs for the changes in this PR so will add a note about that. Good call out. 👍

Reply:
Oops, didn't refresh before posting so missed the other comments. `IgnoredNamespaces` is also used when creating the support package.

Reply:
Once Ava's changes go in, I'm assuming IgnoredNamespaces will only be used in the support package?

Reply:
Actually nevermind, I see at least two other places where it's used.